Refs #35591 -- Emphasized that runserver is not suitable for production. (#18457)
django/django: docs/howto/deployment/index.txt

Django, being a web framework, needs a web server in order to operate. And
since most web servers don't natively speak Python, we need an interface to

> make that communication happen. The :djadmin:`runserver` command starts
> a development server, but this is not suitable for production.

Reviewer: ```suggestion
make that communication happen. The :djadmin:`runserver` command starts a
lightweight development server, which is not suitable for production.
```
or "that is" maybe
